include/mbgl/shaders/shader_defines.hpp

```diff
@@ -62,6 +64,11 @@ enum {
     collisionTextureCount
 };
 
+enum {
```

Comment:
Prefer `enum class`.

Reply:
Everything else in the file uses `enum`. I would prefer keeping them the same so it's either updating all of them to `enum class` now or later.

Reply:
Created an issue.